openpype/hosts/nuke/api/pipeline.py

```diff
@@ -217,7 +219,8 @@ def _install_menu():
 
     if not ASSIST:
         label = "{0}, {1}".format(
-            os.environ["AVALON_ASSET"], os.environ["AVALON_TASK"]
+            get_current_asset_name(),
+            get_current_task_name()
```

Comment:
Should this also get a `get_context_label()` function like you implemented for some others?

Reply:
Done